proto/services/tasks_svc/v1/task_template_svc.proto

```diff
@@ -0,0 +1,184 @@
+syntax = "proto3";
+
+package services.tasks_svc.v1;
+
+option go_package = "gen/proto/services/tasks-svc";
+
+///
+/// CreateTaskTemplate
+///
+
+message CreateTaskTemplateRequest {
+  string name = 1; // @gotags: validate:"required"
+  optional string description = 2;
+  optional string ward_id = 4;
+  repeated SubTask subtasks = 5; // When passed, every subtask is a seperate entity
+
+  message SubTask {
+    string name = 1;
+  }
+}
+
+message CreateTaskTemplateResponse {
+  string id = 1;
+}
+
+
+///
+/// GetAllTaskTemplates
+///
+
+message GetAllTaskTemplatesRequest {
```

Comment:
do we not need a per-ward query? not suggesting anything, just wondering

Reply:
Obviously not, same protos as in task-svc